client/jetpack-cloud/sections/pricing/jpcom-masterbar/style.scss

```diff
@@ -737,85 +740,93 @@
 		animation-delay: calc(calc(0.085s * 7) + calc(0.35s / 1.5));
 	}
 
-	.header__submenu-bundles {
-		hr,
-		.header__submenu-bundles-heading,
-		.header__submenu-links-list {
-			max-width: 1200px;
-			width: 85%;
-			margin-left: auto;
-			margin-right: auto;
+	.header__submenu-bottom-section {
+		@media ( min-width: 900px ) {
+			display: flex;
+			gap: 6rem;
 		}
+	}
 
-		hr {
-			background-color: var(--studio-gray-10);
-			margin-bottom: 1.125rem;
-		}
+	.header__submenu-bundles {
+		flex: 2;
+	}
 
-		.header__submenu-bundles-heading {
-			color: var(--studio-gray-50);
-			font-size: 1.25rem;
-			margin-bottom: 0;
-		}
+	.header__submenu-apps-wrapper {
+		flex: 1;
 
-		.header__submenu-links-list {
-			display: flex;
-			justify-content: flex-start;
-			gap: 2rem;
-
-			> li {
-				flex: 0;
-				min-width: 200px;
-				animation-fill-mode: backwards;
-				animation-name: submenu-item;
-				animation-duration: 0.25s;
-				animation-timing-function: ease-in-out;
-				animation-delay: calc(calc(0.085s * 8) + calc(0.35s / 1.5));
-			}
+		.header__submenu-link {
+			margin-top: 1.75rem;
+			animation-delay: calc(calc(0.085s * 9) + calc(0.35s / 1.5));
 		}
+	}
 
-		@media ( prefers-reduced-motion ) {
-			.header__submenu-links-list > li {
-				animation-duration: 0;
-				animation-delay: 0;
-			}
+	.header__submenu-category-heading {
+		color: var(--studio-gray-50);
+		font-size: 1.25rem;
+		margin-bottom: 0;
+	}
+
+	.header__submenu-section-separator {
+		background-color: var(--studio-gray-10);
+		margin-bottom: 1.125rem;
+
+
+		@media ( max-width: 782px ) {
+			margin-left: 2rem;
+			margin-right: 2rem;
+			width: auto;
 		}
+	}
+
+	.header__submenu-section-separator,
+	.header__submenu-bottom-section {
+		max-width: 1200px;
+		width: 85%;
+		margin-left: auto;
+		margin-right: auto;
 
 		@media ( max-width: 1300px ) {
-			.header__submenu-bundles-heading,
-			.header__submenu-links-list,
-			hr {
-				width: 90%;
-			}
+			width: 90%;
 		}
 		@media ( max-width: 1100px ) {
-			.header__submenu-bundles-heading,
-			.header__submenu-links-list,
-			hr {
-				width: 95%;
-			}
+			width: 95%;
 		}
+	}
+
+	.header__submenu-bundles .header__submenu-links-list {
+		display: flex;
+		justify-content: flex-start;
+		gap: 2rem;
+
 		@media ( max-width: 900px ) {
-			.header__submenu-links-list {
-				display: block;
+			display: block;
+		}
+
+		> li {
+			flex: 0;
+			min-width: 200px;
+			animation-fill-mode: backwards;
+			animation-name: submenu-item;
+			animation-duration: 0.25s;
+			animation-timing-function: ease-in-out;
+			animation-delay: calc(calc(0.085s * 8) + calc(0.35s / 1.5));
+
+			@media ( prefers-reduced-motion ) {
+				animation-duration: 0;
+				animation-delay: 0;
 			}
 
-			.header__submenu-links-list > li {
+			@media ( max-width: 900px ) {
 				animation: none;
 			}
 		}
-		@media ( max-width: 782px ) {
-			.header__submenu-bundles-heading,
-			.header__submenu-links-list {
-				padding: 0 2rem;
```

Comment:
Not sure if this is the line that's affecting it, but the padding in the bottom section of the menu is off on mobile
![Screen Shot 2023-01-23 at 4 54 00 PM](https://user-images.githubusercontent.com/18016357/214160202-b411c7b2-9135-4c80-8faa-86a68959b746.png)